mirascope/core/anthropic/call_response.py

```diff
@@ -70,6 +76,16 @@ def finish_reasons(self) -> list[str]:
         """Returns the finish reasons of the response."""
         return [str(self.response.stop_reason)]
 
+    @property
+    def tool_calls(self) -> list[ToolUseBlock] | None:
```

Comment:
as discussed, I don't think we should add this (and instead we should remove the tool_calls property from all call responses and just have tools/tool properties)

Reply:
done